components/fsxa/sections/InteractiveYoutubeVideo.tsx

```diff
@@ -0,0 +1,74 @@
+import { Component } from 'vue-property-decorator'
+
+import { RichTextElement } from 'fsxa-api'
+import { FSXABaseSection } from 'fsxa-pattern-library'
+import { Headline, Container, LineSeparator, TeaserSection } from 'fsxa-ui'
+
+export interface Payload {
+  st_autoPlay: boolean
+  st_catalog: []
+  st_interactiveVideo: {
+    fsType: string
+    name: string
+    value: {}
+  },
+  st_youtubeVideo: {
+    fsType: string,
+    name: string,
+    dapType: string,
+    value: [
+      {
+        fsType: string
+        identifier: string
+        url: string
+        value: {
+          description: RichTextElement[]
+          id: string
+          posterUrl: string
+          thumbnailUrl: string
+          title: string
+        }
+      }
+    ]
+  }
+}
+export interface Meta {
+  route: string
+}
+@Component({
+  name: 'InteractiveYouTubeVideo'
+})
+class InteractiveYouTubeVideo extends FSXABaseSection<Payload, Meta> {
+
+  render() {
+    const youtubeID = this.payload.st_youtubeVideo.value[0].identifier;
+    const autoplayParam = this.payload.st_autoPlay.valueOf() == true ? "?autoplay=true" : "";
+    const title = this.payload.st_youtubeVideo.value[0].value.title;
+    const description = this.payload.st_youtubeVideo.value[0].value.description;
+    const url = "https://www.youtube-nocookie.com/embed/"
+    return (
+
+      <div
+        data-testid="fsxa-video-section">
```

Comment:
This `div`-Element is redundant. 

And is the testid necessary at the moment? 
If not please remove it. If it is necessary you could add it to the `Container` element.

Reply:
The <div> was there because previously this returned two alternatives... indeed it is now redundant.

Reply:
Funny, the `div` in my previous comment is interpreted as html.